packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -560,6 +560,73 @@ function AsyncSortTable() {
 }
 ```
 
+## Column Resizing <VersionBadge version="beta" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+TableView supports resizable columns, allowing users to dynamically adjust the width of a column. To designate that a Column is resizable, provide it with the `allowsResizing` prop. This will render a draggable
+resizer handle for each resizable column that becomes visible on hover. Keyboard, touch, and screen reader users can enter "resize" mode by interacting with the target column's header and selecting the "Resize column" option
+from the dropdown.
+
+### Width values
+
+Columns support four different width props: `defaultWidth`, `width`, `minWidth`, and `maxWidth`. Each of these props accepts fixed values or percentages, with `width` and `defaultWidth` accepting fractional (`fr`) units as well.
+Providing a non-fractional value `width` to a Column sets its width to the value provided to it, making it unaffected by any changes to other columns' widths. Providing a `defaultWidth` allows you
+to specify a Column's initial width, but allows the column width to freely shrink and grow in response to changes in other column widths. `minWidth` and `maxWidth` allows you to restrict a Column's width to a minimum and
+maximum value respectively. In general, TableView prioritizes columns with defined widths and divides the remaining space evenly amongst the other columns.
+
+### Resize events
+
+TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's in a controlled fashion. TableView also accepts
```

Comment:
```suggestion
TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's width in a controlled fashion. TableView also accepts
```